modules/client/src/main/java/org/apache/ignite/internal/client/sql/ClientSql.java

```diff
@@ -253,20 +251,6 @@ public <T> CompletableFuture<AsyncResultSet<T>> executeAsync(
         return ch.serviceAsync(ClientOp.SQL_EXEC, payloadWriter, payloadReader);
     }
 
-    /** {@inheritDoc} */
-    @Override
-    public ReactiveResultSet executeReactive(@Nullable Transaction transaction, String query, @Nullable Object... arguments) {
-        // TODO IGNITE-17058.
-        throw new UnsupportedOperationException("Not implemented yet.");
-    }
-
-    /** {@inheritDoc} */
-    @Override
-    public ReactiveResultSet executeReactive(@Nullable Transaction transaction, Statement statement, @Nullable Object... arguments) {
```

Comment:
does it make sense to remove `ReactiveResultSet` interface as well?

Reply:
Good catch, done.